common/src/leap/soledad/common/couch.py

```diff
@@ -1474,14 +1506,32 @@ def ensure_database(self, dbname):
         databases. But, in case it ever does, by raising an exception here we
         have one more guarantee that no modified client will be able to
         enforce creation of a database when syncing.
+        This can only be done by using a shell script with proper validation
+        and permissions (create_cmd).
```

Comment:
Since we have a bigger one on README, makes sense to remove it here.
You mean all of it or just this diff? I can't say if it is clear, I am biased because I wrote the code and that makes the text feel simple to me.